### DISCUSSION TITLE
Index.html works, nothing else does

### DISCUSSION THREAD

**Discussion by razzlejazzled** [community/community#22240] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, having an issue with github pages deployment. Site in question: <a href="https://razzlejazzled.github.io/RJ-G-Portfolio/" rel="noopener nofollow ugc">https://razzlejazzled.github.io/RJ-G-Portfolio/</a>, Repo: <a href="https://github.com/razzlejazzled/RJ-G-Portfolio" rel="noopener nofollow ugc">https://github.com/razzlejazzled/RJ-G-Portfolio</a></p>
<p>Everything works perfectly fine in local host. In pages, the index.html file works just fine, however there is an issue with the links to other pages (down in the selected projects section, the buttons with the plus signs on them). Instead of loading up the details of the project in question, a 404 error occurs. I tried looking through the docs for Github pages, but I either missed something or didn’t find a solution.</p>
<p>I’m still a relative newbie when it comes to coding, but I’ve used pages for a few other projects and never encountered any issues. Thanks in advance for any help or suggestions!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>At a glance, there are a bunch of problems:</p>
<pre><code class="lang-auto">&lt;a href="https://razzlejazzled.github.io/RJ-G-Portfolio/pages/portfolio-details.html"&gt; &lt;span class="ion-ios-plus-outline"&gt;&lt;/span&gt;&lt;/a&gt;
</code></pre>
<p>The directory in your repo is called <code>Pages/</code>, not <code>pages/</code>. You must use the exact same capitalization (that it works locally indicates you’re working on Windows, which ignores capitalization in filenames). Also it’d be better to make the link relative: <code>Pages/portfolio-details.html</code></p>
<pre><code class="lang-auto">&lt;a href="/pages/lplan-details.html"&gt; &lt;span class="ion-ios-plus-outline"&gt;&lt;/span&gt;&lt;/a&gt;
</code></pre>
<p>There’s the same capitalization problem, and in addition the link is absolute, not relative (because of the leading slash). <code>Pages/lplan-details.html</code> should work.</p>
<p>Look through the remaining links for similar issues.</p>

**Comment by razzlejazzled** [comment]
<p>Thanks for taking a look. A bit embarrassed that I missed something like that, but I suppose that’s the benefit of a fresh set of eyes.</p>
<p>Sadly, that suggested changes do not fully fix the problem. Instead of getting the default Github 404 error page as was happening previously, the suggested changes lead to a never ending spinning wheel. Upon opening up the console, the 404 error is still present.</p>

**Comment by airtower-luna** [comment]
<p>The site seems to load just fine for me, as long as Javascript is enabled. I notice your post and the latest commits were made at about the same time, maybe the site just took a moment to update?</p>

**Comment by razzlejazzled** [comment]
<p>Interesting, I’m still getting the same problem. Index.html works just fine, but the links down in the portfolio section lead to 404 errors. Tried it in multiple browsers just to check.</p>

**Comment by airtower-luna** [comment]
<p>Oh, sorry, I misunderstood and thought you were getting the spinning wheel on the main page. On the portfolio pages I get it as well, and that looks like a relative link issue again. For example in <code>Pages/portfolio-details.html</code> you have this:</p>
<pre><code class="lang-auto">  &lt;!-- Vendor CSS Files --&gt;
  &lt;link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet"&gt;
</code></pre>
<p>There is no <code>assets/</code> directory in <code>Pages/</code>, it’s a level above, so the <code>href</code> should be <code>../assets/vendor/bootstrap/css/bootstrap.min.css</code>. Same for a lot of other CSS, image, and script files.</p>

**Comment by razzlejazzled** [comment]
<p>Alright, well I feel sufficiently embarrassed for missing something as simple as that.</p>
<p>It seems to be working, so I thank you for your time and patience helping someone with only 3 months coding experience out with these simple mistakes.</p>
